cmd/kubeadm/app/cmd/phases/init/preflight.go

```diff
@@ -63,15 +63,16 @@ func runPreflight(c workflow.RunData) error {
 		return err
 	}
 
-	if !data.DryRun() {
-		fmt.Println("[preflight] Pulling images required for setting up a Kubernetes cluster")
-		fmt.Println("[preflight] This might take a minute or two, depending on the speed of your internet connection")
-		fmt.Println("[preflight] You can also perform this action in beforehand using 'kubeadm config images pull'")
-		if err := preflight.RunPullImagesCheck(utilsexec.New(), data.Cfg(), data.IgnorePreflightErrors()); err != nil {
-			return err
-		}
-	} else {
+	if data.DryRun() {
 		fmt.Println("[preflight] Would pull the required images (like 'kubeadm config images pull')")
+		return nil
+	}
+
+	fmt.Println("[preflight] Pulling images required for setting up a Kubernetes cluster")
+	fmt.Println("[preflight] This might take a minute or two, depending on the speed of your internet connection")
+	fmt.Println("[preflight] You can also perform this action in beforehand using 'kubeadm config images pull'")
+	if err := preflight.RunPullImagesCheck(utilsexec.New(), data.Cfg(), data.IgnorePreflightErrors()); err != nil {
```

Comment:
```
	if err := preflight.RunPullImagesCheck(utilsexec.New(), data.Cfg(), data.IgnorePreflightErrors()); err != nil {
		return err
	}

	return nil
```
this can be 
```
	return preflight.RunPullImagesCheck(utilsexec.New(), data.Cfg(), data.IgnorePreflightErrors())
```

Reply:
+1